lib/l10n/app_en.arb

```diff
@@ -2633,4 +2633,4 @@
   "@yes": {
     "description": "A positive status"
   }
-}
\ No newline at end of file
+}
```

Comment:
Make sure to remove the newline here so that it passes the CI!

Reply:
Done.

Reply:
@wadamT Your PR now includes a new file, `lib/app_en.arb`. Once you delete that, this should be good to go!

Reply:
@micahmo Done.